### PR TITLE
Android migration guide 2.x to 3.x

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

**Review by bruno-garcia — APPROVED** [review]

File: src/platforms/android/migration.mdx [hunk 1]
```diff
@@ -160,3 +160,17 @@ _New_:
...
+
+```xml
+<application>
+  <meta-data android:name="io.sentry.session-tracking.enable" android:value="false" />
+</application>
+```
+
+**Package changes**
+
+The package `io.sentry.core` has been renamed to `io.sentry`, do a String replace on all the usages from `io.sentry.core.` to `io.sentry.` that it should compile normally.
```
Comment (bruno-garcia) [inline src/platforms/android/migration.mdx:176]:
```suggestion
## Migrating from `sentry-android 2.x` to `sentry-android 3.x`

**Package changes**

The package `io.sentry.core` has been renamed to `io.sentry`, do a String replace on all the usages from `io.sentry.core.` to `io.sentry.` that it should compile normally.

Other than that, the APIs didn't change. Release heath is enabled by default though, if you prefer not to track the health of your releases, disable it with:

```xml
<application>
  <meta-data android:name="io.sentry.session-tracking.enable" android:value="false" />
</application>
```

```

**Review by bruno-garcia — CHANGES_REQUESTED** [review]
Maybe we put the breaking changes first. I honesly missed it when I read "The APIs are pretty much the same" in the start of the section

File: src/platforms/android/migration.mdx [hunk 2]
```diff
@@ -160,3 +160,17 @@ _New_:
 ```java
 Sentry.setExtra("extra", "thing");
 ```
+
+## Migrating from `sentry-android 2.x` to `sentry-android 3.x`
+
+The APIs are pretty much the same, Release heath is enabled by default though, to disable it:
```
Comment (MimiDumpling) [inline src/platforms/android/migration.mdx:166]:
```suggestion
The APIs haven't changed. However, Release Health sets by default. 

To disable Release Health default:
```

File: src/platforms/android/migration.mdx (hunk 1 above)
Comment (MimiDumpling) [reply src/platforms/android/migration.mdx:176]: How about this:

The package `io.sentry.core` has been renamed to `io.sentry`. To compile correctly, replace the strings on all the usages from `io.sentry.core.` to `io.sentry`.

Other than that, the APIs didn't change. Release Heath sets by default. If you prefer not to track the health of your releases, disable it with:

**Review by MimiDumpling — APPROVED** [review]
Just a few suggestions. Thanks for updating the docs!

File: src/platforms/android/migration.mdx [hunk 3]
```diff
@@ -160,3 +160,16 @@ _New_:
...
 Sentry.setExtra("extra", "thing");
 ```
+
+## Migrating from `sentry-android 2.x` to `sentry-android 3.x`
+
+**Package changes**
+
+The package `io.sentry.core` has been renamed to `io.sentry`, do a String replace on all the usages from `io.sentry.core.` to `io.sentry.` that it should compile normally.
+
+Other than that, the APIs didn't change. Release heath is enabled by default though, if you prefer not to track the health of your releases, disable it with:
```
Comment (marandaneto) [inline src/platforms/android/migration.mdx:170]:
```suggestion
The package `io.sentry.core` has been renamed to `io.sentry`. To compile correctly, replace the strings on all the usages from `io.sentry.core.` to `io.sentry.`

Other than that, the APIs didn't change. Release Heath sets by default. If you prefer not to track the health of your releases, disable it with:
```

**Review by marandaneto — COMMENTED** [review]
thanks for reviewing it @MimiDumpling :)

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**